components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -116,6 +117,7 @@ AtmosphereOutput (const ekat::Comm& comm, const ekat::ParameterList& params,
  : m_comm         (comm)
  , m_add_time_dim (true)
 {
+  params.print();
```

Comment:
This (and the ASD print statements below) should probably be removed before merging.